tests/phpunit/integration/Modules/AnalyticsTest.php

```diff
@@ -153,6 +153,62 @@ public function test_get_datapoints() {
 		);
 	}
 
+	/**
+	 * @dataProvider data_parse_date_range
+	 */
+	public function test_parse_date_range( $period_requested, $previous_period_end_offset ) {
+		$analytics = new Analytics( new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
+
+		$result = $analytics->parse_date_range( 'last-' . $period_requested . '-days', 1, 1, true, true );
+		$previous_end = strtotime( $result[ 1 ] );
+		$yesterday_day_of_week = gmdate( 'w', strtotime( 'yesterday' ) );
+		$diff = round( ( strtotime( 'yesterday' ) - ( $period_requested * 86400 )  - $previous_end ) / 86400 );
```

Comment:
Let's use the WP constant instead of the literal here for more clarity.
```suggestion
		$diff = round( ( strtotime( 'yesterday' ) - ( $period_requested * DAY_IN_SECONDS )  - $previous_end ) / DAY_IN_SECONDS );
```

Reply:
Oh, right!